Python/perf_trampoline.c

```diff
@@ -253,7 +253,7 @@ perf_map_write_entry(void *state, const void *code_addr,
                                   NULL);
         return;
     }
-    fprintf(method_file, "%p %x py::%s:%s\n", code_addr, code_size, entry,
+    fprintf(method_file, "%llx %x py::%s:%s\n", code_addr, code_size, entry,
```

Comment:
What if it's a 32-bit machine? Python still supports that https://www.python.org/downloads/release/python-3120a7/

Also, the compiler reports a lot of warnings. This should be fixed.